tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -446,6 +451,104 @@ def GraphExecutionData(self, run, begin, end, trace_id=None):
             ],
         }
 
+    def GraphOpInfo(self, run, graph_id, op_name):
+        """Get the information regarding a graph op's creation.
+
+        Args:
+          run: Name of the run.
+          graph_id: Debugger-generated ID of the graph that contains
+            the op in question. This ID is available from other methods
+            of this class, e.g., the return value of `GraphExecutionDigests()`.
+          op_name: Name of the op.
+
+        Returns:
+          A JSON-serializable object containing the information regarding
+          the op's creation.
+
+        Raises:
+          NotFoundError if the graph_id or op_name does not exist.
+        """
+        runs = self.Runs()
```

Comment:
nit: variable used only once; could be inlined

Reply:
For that matter, do you even need this test?  What situation is it protecting you from?

Reply:
OK.  Partly I guess I was wondering if this is the right place for the test (as opposed to doing it first in the caller) but nbd.  TODO SGTM, but I don't see it?

Reply:
Whoops. Added it.